bitnami/postgresql-ha/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 10.0.3
+version: 10.0.4
```

Comment:
Could you please bump the Chart version again? Version `10.0.4` has already been released
https://github.com/bitnami/charts/blob/main/bitnami/postgresql-ha/Chart.yaml